src/index.js

```diff
@@ -1,10 +1,11 @@
-export { default as Text } from './Text'
+export { default as BackgroundImage } from './BackgroundImage'
 export { default as Box } from './Box'
 export { default as Flex } from './Flex'
-export { default as Image } from './Image'
-export { default as BackgroundImage } from './BackgroundImage'
-export { default as Icon } from './Icon'
 export { default as Heading } from './Heading'
+export { default as Icon } from './Icon'
+export { default as Image } from './Image'
+export { default as Link } from './Link'
+export { default as Text } from './Text'
```

Comment:
Alphabetical order? Might make merge conflicts here a little easier to manage 👍

Reply:
Yes for Alphabetical order if it feels right to the contributors. Definitely less risk for merge conflicts & potentially less code diff as well.